english/Utilize-your-wand-once-again/crappy-wand-spell.md

```diff
@@ -0,0 +1,11 @@
+Letting out a light sigh, you once grip your makeshift wand firmly and concentrate.
+Gathering your focus, you point your rope and fork wand to the door and firmly encant "Alohomora!"
+
+		The door begins to acquire a dim yellow-green glow emanating from the lock mechanism.
+		Rattling sounds out from the door's keyhole as the door begins to shudder, wood
+		splintering and shuddering as a resounding grinding of metal and wood fill the air.
+
+							The door jars opens with a creak 
+							as the locking mechanism breaks.
+
+[The door is open! Is there something else I forgot?](../../marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.